src/pages/Product/Product.js

```diff
@@ -1,9 +1,178 @@
 import React, { Component } from 'react';
 
+// import { GET_PRODUCTS_API } from '../../../src/config.js';
+
+import ProductCard from './ProductCard/ProductCard';
+
+import './Product.scss';
+
 export class Product extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productCard: [],
+      addCart: false,
+      bone: false,
+      hair: false,
+      growth: false,
+      skin: false,
+    };
+  }
+
+  componentDidMount() {
+    // fetch(`${GET_PRODUCTS_API}`)
+    fetch('./data/ProductData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productCard: data.message,
+        });
+      });
+  }
+
+  makeCondition = () => {
+    let fillter = [];
+
+    if (this.state.bone) {
+      fillter.push(`efficacy=1`);
+    }
+
+    if (this.state.hair) {
+      fillter.push(`efficacy=2`);
+    }
+
+    if (this.state.growth) {
+      fillter.push(`efficacy=3`);
+    }
+
+    if (this.state.skin) {
+      fillter.push(`efficacy=4`);
+    }
+
+    // fetch(`${GET_PRODUCTS_API}?`)
+    fetch(`./data/ProductData.json?${fillter.join('&')}`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productCard: data.message,
+        });
+      });
+  };
+
+  handleCheckBox = event => {
+    this.setState(
+      {
+        [event.target.name]: !this.state[event.target.name],
```

Comment:
event.target.name을 2번정도 사용하고 있는데 이런건 하나 변수 선언해서 거기에 할당해두고 쓰면 두 값이 동일한 값이다 라는 것도 표현되고 가독성도 좋을 것 같습니다.
```suggestion
    const checkBoxName = event.target.name
    this.setState(
      {
        [checkBoxName]: !this.state[checkBoxName],
```

Reply:
오 그렇네요. 저는 event.target.name가  []에 들어간다는것 자체를 찾기까지가 오래걸려서 미처 멀리까지 내다보지 못했어요 😂